strawberryfields/ops.py

```diff
@@ -1799,17 +1797,17 @@ class Gaussian(Preparation, Decomposition):
             If None, it is assumed that :math:`r=0`.
         decomp (bool): Should the operation be decomposed into a sequence of elementary gates?
             If False, the state preparation is performed directly via the backend API.
-        hbar (float): the value of :math:`\hbar` used in the definition of the :math:`\x`
-            and :math:`\p` quadrature operators. Note that if used inside of an engine
-            context, the hbar value of the engine will override this keyword argument.
         tol (float): the tolerance used when checking if the matrix is symmetric: :math:`|V-V^T| \leq tol`
     """
     # pylint: disable=too-many-instance-attributes
     ns = None
 
-    def __init__(self, V, r=None, decomp=True, hbar=2, tol=1e-6):
+    def __init__(self, V, r=None, decomp=True, tol=1e-6):
         # TODO NOTE: there is no contextual hbar value anymore, is the hbar actually necessary here?
-        self.hbar = hbar
+
+        #: float: value of :math:`\hbar` used in the definition of the :math:`\x` and :math:`\p` quadrature operators
+        # TODO can we just divide V by hbar/2 here and remove hbar from further expressions?
```

Comment:
Probably, I think

Reply:
Maybe I'll give it a go later, but for now it ought to work.